templates/components/common/navigation-menu.html

```diff
@@ -16,7 +16,7 @@
         {{#unless theme_settings.hide_content_navigation}}
             {{#each pages}}
                  <li class="navPages-item navPages-item-page">
-                     <a class="navPages-action" href="{{url}}">{{name}}</a>
+                     <a class="navPages-action{{#if name '==' ../page.title}} activePage{{/if}}" href="{{url}}">{{name}}</a>
```

Comment:
why this check ?

Reply:
This is for the parent-page scenario. If a page is nested in a tree this will highlight both the current page and its parent, like in: https://user-images.githubusercontent.com/1546172/45905025-c491ab80-bda3-11e8-8302-fab27866a954.png